frontend/src/lib/components/ui/heatmap/heatmap.svelte

```diff
@@ -21,34 +20,27 @@
     weekday: 'long',
   });
 
-  const dayOfWeekLabels = Array.from(
-    { length: 7 },
-    (_, i) => {
-      const date = new Date();
-      date.setDate(date.getDate() - date.getDay() + i + 1); // we + 1 to make 0 = monday
-      return dayOfWeekFormatter.format(date);
-    }
-    // dayOfWeekFormatter.format(new Date(2024, 8, i)) // Month that starts on a Monday (could be any - just for sorting)
-  );
+  const dayOfWeekLabels = Array.from({ length: 7 }, (_, i) => {
+    const date = new Date();
+    date.setDate(date.getDate() - date.getDay() + i + 1); // we + 1 to make 0 = monday
```

Comment:
Is the match not still incorrect here?

Reply:
Yes, but I updated the code to be more readable IMO